src/gamemode.c

```diff
@@ -535,6 +535,21 @@ handle_unregister_game_by_pidfd (XdpDbusGameMode *object,
   return G_DBUS_METHOD_INVOCATION_HANDLED;
 }
 
+static void
+client_properties_changed (GDBusProxy* proxy,
+                           GVariant* changed_properties,
+                           char** invalidated_properties)
```

Comment:
The code style of the project places the asterisks together with the variable name, not the type. E.g. `char **invalidated_properties`.

Reply:
Ah thank you. Fixed.